src/utils/number.ts

```diff
@@ -8,3 +8,23 @@ export const getAccelerate = (acceleration: number, time: number) => {
     ? Math.ceil(acceleration * time * 100) / 100
     : 0;
 };
+
+//string타입 변수를 숫자로 변환하는 함수
+export const stringToNumber = (number: string) => parseInt(number.replace(',', ''));
+
+//숫자를 3자리마다 ,로 표현되는 string으로 변환
+export const numberToString = (number: number) => {
+  let str = '';
+  let count = 0;
+  while (number > 0) {
+    if (count === 3) {
+      str += ',';
+      count = 0;
+    }
+    let digit = number % 10;
+    str += digit;
+    number = Math.floor(number / 10);
+    count += 1;
+  }
+  return str.split('').reverse().join('');
+};
```

Comment:
이런 방식도 있어 ㅎㅎ....

```suggestion
export const addCommaInNumber = number => {
  return number.toLocaleString('ko-KR');
};
```

함수 이름도 조금 더 생각해보는 게 어떨까!? 👍 


https://hianna.tistory.com/441
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

Reply:
아 이런게 있구나 함수 이름도 좀 더 생각해볼겡